SLAM/EKFSLAM/ekf_slam.py

```diff
@@ -112,16 +108,12 @@ def calc_n_lm(x):
 
 
 def jacob_motion(x, u):
-    Fx = np.hstack((np.eye(STATE_SIZE), np.zeros(
-        (STATE_SIZE, LM_SIZE * calc_n_lm(x)))))
```

Comment:
This function is only called with `x` equal to the robot pose, so `calc_n_lm(x)` is 0 and `Fx` is the identity matrix. This change makes the function simpler but removes the capability to pass state including land marks to the function, which is not used.

Reply:
I would like to keep the code same as before, the reason is same as upper my answer.

Reply:
I see your point, thanks for explaining. I undid those commits now and only kept the changes from elementwise to matrix multiplication.